.github/workflows/build-images.yml

```
@@ -51,14 +57,16 @@ jobs:
         env:
           APP_ID: ${{ matrix.environment.app_id }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           DEPLOY_AWS_ACCESS_KEY_ID: ${{ secrets[matrix.environment.aws_deploy_key] }}
           DEPLOY_AWS_SECRET_ACCESS_KEY: ${{ secrets[matrix.environment.aws_deploy_secret] }}
-          AWS_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
-          SENTRY_DSN: ${{ secrets.SENTRY_TOKEN }}
-          PERM_ENV: ${{ matrix.environment.perm_env }}
+          NEW_RELIC_LICENSE_KEY: ${{ secrets[matrix.environment.new_relic_license_key] }}
+          NEW_RELIC_APPLICATION_NAME: ${{ secrets[matrix.environment.new_relic_application_name] }}
```

This isn't a secret, and so ended up not getting populated:
```suggestion
          NEW_RELIC_APPLICATION_NAME: ${{ matrix.environment.new_relic_application_name }}
```

---

ah sorry.. My bad..

---

No worries, this is why we test! :slightly_smiling_face: